tensorboard/webapp/settings/_views/dialog_container.ts

```diff
@@ -0,0 +1,63 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+import {Component} from '@angular/core';
+import {Store, select} from '@ngrx/store';
+
+import {
+  getReloadEnabled,
+  getReloadPeriodInMs,
+  getPageSize,
+} from '../_redux/settings_selectors';
+import {State} from '../_redux/settings_types';
+import {
+  toggleReloadEnabled,
+  changeReloadPeriod,
+  changePageSize,
+} from '../_redux/settings_actions';
+
+/** @typehack */ import * as _typeHackRxjs from 'rxjs';
+
+@Component({
+  selector: 'settings-dialog',
+  template: `
+    <settings-dialog-component
+      [reloadEnabled]="reloadEnabled$ | async"
+      [reloadPeriodInMs]="reloadPeriodInMs$ | async"
+      [pageSize]="pageSize$ | async"
+      (reloadToggled)="onReloadToggled()"
+      (reloadPeriodInMsChanged)="onReloadPeriodInMsChanged($event)"
+      (pageSizeChanged)="onPageSizeChanged($event)"
+    ></settings-dialog-component>
+  `,
+})
+export class SettingsDialogContainer {
+  readonly reloadEnabled$ = this.store.pipe(select(getReloadEnabled));
```

Comment:
can we just use `... = this.store.select(getReloadEnabled)`?

Reply:
Done.